dev/com.ibm.ws.session/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -136,4 +136,14 @@ extraProperties.name=Additional properties
 extraProperties.description=Additional properties related to session management.
 
 modify.active.count.on.invalidated.session.name=Modify active count on invalidated session
-modify.active.count.on.invalidated.session.description=Httpsession activeCount metric might become inaccurate when the session is accessed by multiple applications. Set the property to "false" to resolve the issue.
\ No newline at end of file
+modify.active.count.on.invalidated.session.description=Httpsession activeCount metric might become inaccurate when the session is accessed by multiple applications. Set the property to "false" to resolve the issue.
+
+allow.serialized.access.name=Allow serialized access
+allow.serialized.access.description=Enable this option to allow serialized access to session data.
+
+max.wait.time.name=Max wait time
+max.wait.time.description=The amount of time, in milliseconds, a servlet waits on a session before it continues execution.
```

Comment:
Change "in milliseconds" to "in seconds" in max.wait.time.description.
The code will convert to millisecond:
long syncSessionTimeOut = (long)_smc.getSerializedSessionAccessMaxWaitTime() * 1000L; //convert to millisecond